src/pages/home/report/ReportActionsView.js

```diff
@@ -133,7 +130,6 @@ class ReportActionsView extends React.Component {
 
     shouldComponentUpdate(nextProps, nextState) {
         if (!_.isEqual(nextProps.reportActions, this.props.reportActions)) {
-            this.sortedAndFilteredReportActions = this.getSortedReportActionsForDisplay(nextProps.reportActions);
             this.mostRecentIOUReportActionID = ReportActionsUtils.getMostRecentIOUReportActionID(nextProps.reportActions);
             return true;
         }
```

Comment:
Is this required in here as well if we have added the logic to the ReportScreen?

Reply:
Yes, the `mostRecentIOUReportActionID` is used just in this component and sends it down, the `ReportScreen` now just is taking care of the sorting, the logic for `mostRecentIOUReportActionID` isn't in `ReportScreen`

Reply:
ohhhh got it, yes it is true, my bad, I miss it after some merge conflicts, updating PR